internal/pkg/config/config_test.go

```diff
@@ -144,13 +145,47 @@ func TestLoadVolumeMountsConfig(t *testing.T) {
 	}
 }
 
+func TestLoadInitSidecarConfig(t *testing.T) {
+	cfg := volumeMounts
+	c, err := LoadInjectionConfigFromFilePath(initSidecar)
+	if err != nil {
+		t.Fatal(err)
+	}
+	expectedName := "init-sidecar"
+	nExpectedContainers := 0
+	nExpectedVolumes := 0
+	nExpectedEnvironmentVars := 0
+	nExpectedVolumeMounts := 0
+	nExpectedInitContainers := 1
+
+	if c.Name != expectedName {
+		t.Fatalf("expected %s Name loaded from %s but got %s", expectedName, cfg, c.Name)
+	}
+	if len(c.Environment) != nExpectedEnvironmentVars {
```

Comment:
Consider writing an expect function so we dont repeat the same if statement every time. `expect(t, c.Environment, "envVars").hasSize(nExpectedEnvironmentVars)` would be more readable.